### PR TITLE
Fix products filters

### PR DESCRIPTION
**Pull request by Iryna-Bzdyr**

### CODE REVIEW

File: src/containers/image-upload-container/image-upload-container.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ const ImageUploadContainer = ({
       <label htmlFor='upload-photo'>
         <input
           style={{ display: 'none' }}
-          accept='image/*'
+          accept='video/*,image/*'
           id='upload-photo'
```
Comment (moran711) [inline src/containers/image-upload-container/image-upload-container.js:20]: Why this element have id `upload-photo` if you are using photo and video upload?

File: src/translations/product.translations.js [hunk 2]
```diff
@@ -6,10 +6,10 @@ export const productsTranslations = {
   CLEAR_FILTERS: 'ОЧИСТИТИ ФІЛЬТРИ',
   ADD_PRODUCT: 'ДОДАТИ ПРОДУКТ',
```
Comment (vasylshpak) [inline src/translations/product.translations.js:7]: `CLEAR_FILTERS: 'ОЧИСТИТИ ФІЛЬТРИ',
 ADD_PRODUCT: 'ДОДАТИ ПРОДУКТ',` 

Should those values be uppercase?

**Review by vasylshpak — CHANGES_REQUESTED** [review]

File: src/translations/product.translations.js [hunk 3]
```diff
@@ -6,10 +6,10 @@ export const productsTranslations = {
...
   COMPACT_TABLE: 'Компактна таблиця',
-  CATEGORIES: 'КАТЕГОРІЇ',
-  MODELS: 'МОДЕЛІ',
-  PATTERNS: 'ГОБЕЛЕНИ',
-  COLORS: 'КОЛЬОРИ',
+  CATEGORIES: 'Категорії',
+  MODELS: 'Моделі',
+  PATTERNS: 'Гоболени',
+  COLORS: 'Кольори',
```
Comment (nadiyafomenko) [inline src/translations/product.translations.js:12]: typo in word "гобелени"

**Review by vasylshpak — APPROVED** [review]
Good job!

**Review by nadiyafomenko — APPROVED** [review]

**Review by moran711 — APPROVED** [review]

**Pull request merged**